data/json/npcs/exodii/exodii_merchant_talk.json

```diff
@@ -0,0 +1,711 @@
+    "responses": [ { "text": "[Tell them]", "topic": "TALK_EXODII_MERCHANT_Talk_Intro2" } ]
+  },
+  {
+    "id": "TALK_EXODII_MERCHANT_Talk_Intro1b",
+    "type": "talk_topic",
+    "dynamic_line": "Thank ye.  When you an' Rubik first met, howw long'd it been since the mess begun?",
+    "responses": [ { "text": "[Tell them]", "topic": "TALK_EXODII_MERCHANT_Talk_Intro2" } ]
+  },
+  {
+    "id": "TALK_EXODII_MERCHANT_Talk_Intro2",
+    "type": "talk_topic",
+    "dynamic_line": "Lovely!  Benzete's gonna be a pair, that's a track!  I mean, they'll be right happy.  This is their first jump, and it's a flappin' duck.",
+    "speaker_effect": { "effect": { "u_add_var": "rubik_intro", "type": "dialogue", "context": "midpoint", "value": "yes" } },
+    "responses": [
+      { "text": "What makes it such a good jump?", "topic": "TALK_EXODII_MERCHANT_Talk_Intro3" },
+      { "text": "Who is Benzete?", "topic": "TALK_EXODII_MERCHANT_Talk_IntroBenzete1" },
+      { "text": "Can we talk about something else?", "topic": "TALK_EXODII_MERCHANT_Talk" },
+      { "text": "I've gotta go.  See you later, Rubik.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_EXODII_MERCHANT_Talk_Intro2a",
+    "type": "talk_topic",
+    "dynamic_line": "So 'twere an I recalt, us're yarkin' of Benzete an' their great first jump.",
+    "responses": [
+      { "text": "What makes it such a good jump?", "topic": "TALK_EXODII_MERCHANT_Talk_Intro3" },
+      { "text": "Who is Benzete?", "topic": "TALK_EXODII_MERCHANT_Talk_IntroBenzete1" },
+      { "text": "Can we talk about something else?", "topic": "TALK_EXODII_MERCHANT_Talk" },
+      { "text": "I've gotta go.  See you later, Rubik.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_EXODII_MERCHANT_Talk_Intro3",
+    "type": "talk_topic",
+    "dynamic_line": "Us gotta leave the old world afore it's worn right to the gums, an y'ken, and find a nice twitchy one on t'other side.  Your deadland be a brand shiny",
+    "responses": [
+      { "text": "I'm not following that at all.", "topic": "TALK_EXODII_MERCHANT_Talk_Intro3a" },
+      {
+        "text": "[int 9] So you mean, we have a nice, freshly dead world for you to visit?",
+        "condition": { "u_has_intelligence": 9 },
+        "topic": "TALK_EXODII_MERCHANT_Talk_Intro4b"
+      },
+      { "text": "Who is Benzete?", "topic": "TALK_EXODII_MERCHANT_Talk_IntroBenzete1" },
+      { "text": "Can we talk about something else?", "topic": "TALK_EXODII_MERCHANT_Talk" },
+      { "text": "I've gotta go.  See you later, Rubik.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_EXODII_MERCHANT_Talk_Intro3a",
+    "type": "talk_topic",
+    "dynamic_line": "Neverken.  Right and so, the enemy is new here, and us'n have a few good hours in the sun beforenext the jump.",
+    "responses": [
+      {
+        "text": "Oh, you mean you got here early in the cataclysm.  Why is that so good?",
+        "topic": "TALK_EXODII_MERCHANT_Talk_Intro3"
```

Comment:
This seems like a infinite loop, maybe this one should be TALK_EXODII_MERCHANT_Talk_Intro4a ? 
TALK_EXODII_MERCHANT_Talk_Intro4a is not used at all.

Reply:
I did indeed. Thanks, will catch that on the next pass.